lib/src/charts/widgets/bar_chart.dart

```diff
@@ -150,23 +116,14 @@ class _BarChartState extends State<BarChart> with TickerProviderStateMixin {
       controller: _controller,
       width: widget.chartWidth,
       height: widget.chartHeight,
+      uMaxYValue: widget.maxYValue,
```

Comment:
What does `uMaxYValue` mean?

Reply:
Values provided by the user, so the `u` at the start. Right now the min/max values in our controller are calculated minimum's and maximum's